packages/icon/src/Icon/Icon.tsx

```diff
@@ -64,7 +64,12 @@ const Icon = <T extends TgphElement>({
       {...props}
     >
       {IconComponent && (
-        <IconComponent aria-label={alt} width="100%" height="100%" />
+        <IconComponent
+          aria-label={alt}
+          width="100%"
+          height="100%"
+          display="block"
```

Comment:
When the parent container for an icon is inline-block you always need this so the icon acts appropriately in the container